app/frontend/stylesheets/pages/partials/_event-card.scss

```diff
@@ -0,0 +1,76 @@
+#event-card-container {
+  height: 500px;
+  overflow-y:scroll;
+
+}
+
+.event-cards {
+  display: flex;
+  margin-left: 30px;
+}
+
+.event-cards-content {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  background: rgba(227, 225, 225, 0.25);
+  border-radius: 15px;
+  padding: 15px;
+  margin-bottom: 15px;
+}
+
+.event-cards-content:hover {
+  background: rgba(132, 130, 130, 0.25);
+}
+
+.event-cards-content h3 {
+  font-size: 22px;
+  padding-right: 20px;
+}
+
```

Comment:
I have just learned this info, the power of SCSS is that we can nest the different selectors under one selector.
For example, in your `_event.html.erb` you have a section with the id `#event-card-container`. It means that you can add the properties to your `#event-card-container` and underneath you can nest the following selectors: `.event-cards` and `.event-cards-content`. Also as you can nest under `.event-cards-content` the h3 element and :hover property.
```
.event-cards-content {

  &:hover {
  }

  h3 {
  }
}
```
you can look at the `_calendar.scss` partial

Reply:
Really useful feedback, thank you! I didn't know that about scss, I appreciate you sharing.